zio-http-gen-sbt-plugin/src/main/scala/zio/http/gen/sbt/Format.scala

```diff
@@ -0,0 +1,11 @@
+package zio.http.gen.sbt
+
+sealed trait Format
+object Format {
+  case object YAML extends Format
+  case object JSON extends Format
+  def fromFileName(fileName: String): Format =
+    if (fileName.endsWith("yml") || fileName.endsWith("yaml")) YAML
+    else if (fileName.endsWith("json")) JSON
+    else throw new Exception(s"Unknown format for file $fileName")
```

Comment:
pretty naive - if needed, this can be enhanced by simply trying to parse as json, and then yaml. or even use apache-tika to detect file's mimetype.
For now, this seems good enough to me.

Reply:
okay for now